### PR TITLE
Warn for missing / obsolete error docs

### PR DESCRIPTION
**Pull request by Narretz**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Chore

**What is the current behavior? (You can also link to an open issue here)**
We don't know when error docs are missing or obsolete

**Does this PR introduce a breaking change?**
No

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit-message-format

**Other information**:
I have removed an obsolete animation error and moved the tpload error to the correct namespace. In the $templateRequest file, it looks like the error is created in the $compile namespace, but it actually ends up as the $templateRequest namespace, where it should be.

### CODE REVIEW

File: src/ng/templateRequest.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var $templateRequestMinErr = minErr('$compile');
+var $templateRequestMinErr = minErr('$templateRequest');
```
Comment (gkalpak) [inline src/ng/templateRequest.js]: And no test broke? Seems like we're missing one :smiley:

**Review by gkalpak — APPROVED** [review]
I'm no dgeni guru, but afaik LGTM :smiley:
<sub>(It would be nice to have a test for `templateRequestMinErr`, but not a blocker.)</sub>

💯

File: src/ng/templateRequest.js (hunk 1 above)
Comment (Narretz) [inline src/ng/templateRequest.js]: Actually a lot of tests broke. 😨 I didn't think of implications of the change. I think we said that changing the error name/ namespace is a breaking change so we can't do that in 1.6.

**Comment by Narretz** [comment]
I think this is actually an issue with min Err / error extraction task / script: - The min error for $compile:tpload gets extracted as $templateRequest:tpload to errors.json which means the erro match script gets thrown off

**Comment by gkalpak** [comment]
We probably shouldn't use misleading namespaces (var name !== namespace).
We could fix it in 1.7.x

**Pull request merged**